vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/VssDefinitionProcessor.kt

```diff
@@ -141,7 +141,7 @@ class VssDefinitionProcessor(
         private const val PACKAGE_NAME = "org.eclipse.kuksa.vss"
         private const val FILE_NAME_PROCESSOR_POSTFIX = "Processor"
         private const val ASSETS_BUILD_DIRECTORY = "intermediates/assets/"
-        private const val BUILD_FOLDER_NAME = "build/"
+        private const val BUILD_FOLDER_NAME = "/build/"
```

Comment:
This logic won't work on a Windows Runner due to different fileSeparators ("/" vs "\")